### PR TITLE
Config with pure python with multiprocess

### PR DESCRIPTION
**Pull request by icemac**
I created a skeleton for testing buildout recipes and other packages using multiple processes.

Closes #6. (supersedes it)

Built by zopefoundation/meta#35.

### CODE REVIEW

File: src/z3c/recipe/tag/__init__.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def _build_idutils(self):
             ], [
                 '-m',
                 pkg_resources.resource_filename(
-                    "z3c.recipe.tag", "id-lang.map"),
+                    "z3c.recipe.tag", "id-lang.map.txt"),
```
Comment (mgedmin) [inline src/z3c/recipe/tag/__init__.py:139]: This is an interesting change.

May I ask why?

File: tox.ini [hunk 2]
```diff
@@ -37,3 +31,31 @@ commands =
...
+exclude_lines =
+    pragma: nocover
+    except ImportError:
+    raise NotImplementedError
+    if __name__ == '__main__':
+    self.fail
+    raise AssertionError
+
+[coverage:html]
+directory = htmlcov
```
Comment (mgedmin) [inline tox.ini:61]: You're splitting coverage's configuration across two files: `.coveragerc` and here.  I don't like that.  I'm not sure coverage.py even supports this.

**Review by mgedmin — CHANGES_REQUESTED** [review]
Please move coverage.py configuration into a single file, then it'll be fine to merge.

File: tox.ini (hunk 2 above)
Comment (icemac) [reply tox.ini]: I fixed it, it is not supported and it was a mistake on my side.

File: src/z3c/recipe/tag/__init__.py (hunk 1 above)
Comment (icemac) [reply src/z3c/recipe/tag/__init__.py:139]: The reason is not to have a separate rule in `MANIFEST.in` for this one project. As the `MANIFEST.in` is in `meta/config/default` thus shared for all packages.

**Review by mgedmin — APPROVED** [review]

**Comment by icemac** [comment]
Thank you. I am going to rename the template so the script generated #9.

**Pull request closed without merging**